Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/view_routes_unittest.py

```diff
@@ -116,6 +116,7 @@ def test_commit_link(self, driver, **kwargs):
     @WaitForDockerTestCase.mock_if_no_docker(mock_redis=FakeStrictRedis, mock_cassandra=MockCassandraContext)
     @FlaskTestCase.run_with_webserver()
     def test_constants(self, client, **kwargs):
+        self.maxDiff = None
```

Comment:
We probably don't want this committed. (Though I too am annoyed that it isn't None by default!)